### PR TITLE
add modern chair project #87

### PR DESCRIPTION
**Pull request by KRRISHSINGH08**
Please check issue #87 and merge under hacktoberfest. 
Thank You.

### CODE REVIEW

File: Modern Chair Project/index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Modern Chair</title>
+    <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css" />
+	<link rel="stylesheet" href="https://unicons.iconscout.com/release/v2.1.9/css/unicons.css" />
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
```
Comment (0silverback0) [inline Modern Chair Project/index.html:13]: please add a link back to /index.html foe ease of navigation back to the repo landing page

**Review by 0silverback0 — CHANGES_REQUESTED** [review]
Also add you information to the landingPages/index.html card section

File: Modern Chair Project/index.html (hunk 1 above)
Comment (KRRISHSINGH08) [reply Modern Chair Project/index.html:12]: There is no navigation bar in this project. You want me to create one or elaborate please!

**Comment by KRRISHSINGH08** [comment]
> Also add you information to the landingPages/index.html card section
Also, please elaborate more. Do you want me to create a footer for this to add my github and other info there?

**Comment by KRRISHSINGH08** [comment]
> > Also add you information to the landingPages/index.html card section
> > Also, please elaborate more. Do you want me to create a footer for this to add my github and other info there?

**Comment by KRRISHSINGH08** [comment]
@0silverback0  Have a look!

**Comment by 0silverback0** [comment]
please re read the contributing file

**Comment by KRRISHSINGH08** [comment]
Have a look!

**Comment by 0silverback0** [comment]
You haven't added your information to the cards

**Comment by KRRISHSINGH08** [comment]
Added my info. Take a look

**Comment by 0silverback0** [comment]
you need to add this to the LandingPages/index.html  and fill in your PROJECT NAME, YOUR GITHUB PROFILE, YOUR NAME, and the path to YOUR PROJECT DIRECTORY

`    <div class="col-12 col-lg-4 col-md-6">
                  <div class="project bg-black bg-opacity-75 p-4 rounded-5">
                      <p class="fs-5 fw-medium text-white">YOUR PROJECT NAME</p>
                      <p class="d-flex align-items-center gap-2">
                          <i class="fa-brands fa-github text-white"></i>
                          <a href="YOUR GITHUB PROFILE" target="_blank" 
                              class="text-decoration-none text-white">YOUR NAME</a>
                      </p>
                      <div class="w-100 d-flex align-items-center justify-content-end">
                          <button
                              class="d-inline-flex flex-row align-items-center gap-2 px-4 py-3 rounded-pill bg-transparent border border-white">
                              <a href="./YOUR PROJECT DIRECTORY" target="_blank"
                                  class="text-decoration-none text-white d-flex align-items-center gap-2">
                                  <p class="fw-medium">View project</p>
                                  <i class="fa-solid fa-eye"></i>
                              </a>
                          </button>
                      </div>
                  </div>
              </div>`

**Comment by KRRISHSINGH08** [comment]
Done!

**Comment by 0silverback0** [comment]
Steps to make this work. There are 2 directories you need to make changes to. The LandingPages directory's index.html and inside the LandingPages directory there is your modernChair directory

1. run git pull origin main. to get latest changes
2. View The contributing.md file for the sample code
3. update the sample code with your information
4. paste it in the LandingPages/index.html
5. in the ModernChair.index.html make a home link that links to /index.html
6. then push your commit

**Comment by KRRISHSINGH08** [comment]
It's done! Check it please

**Comment by 0silverback0** [comment]
Thank you! Nicely done!

**Pull request merged**